### PR TITLE
Updated screenshots for OCP 3.9

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/4USYdvgB/804-update-screenshots-for-39

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@spadgett @sg00dwin FYI

File: dev_guide/projects.adoc [hunk 1]
```diff
@@ -104,7 +104,7 @@ The right panel shown with the service catalog provides quick access to the most
 recently accessed projects (up to five projects). For the full list of projects,
 use the *View All* link provided at the top of the right panel.
 
-image::3.7-project-list.png[Projects List]
+image::3.9-project-list.jpg[Projects List]
```
Comment (spadgett) [inline dev_guide/projects.adoc]: This looks like the overview instead of project list (if I'm looking at the right image)

**Review by spadgett — COMMENTED** [review]
Thanks @ahardin-rh

File: dev_guide/projects.adoc (hunk 1 above)
Comment (sg00dwin) [reply dev_guide/projects.adoc]: Here's the 3.9 project list 
https://drive.google.com/file/d/1nexOHsTgJ7QorJBq69EzW4ZEhm1gYWzy/view?usp=sharing

Comment (ahardin-rh) [reply dev_guide/projects.adoc]: Whoops! Fixed. Thanks!

**Pull request merged**